indico/htdocs/sass/modules/_attachments.scss

```diff
@@ -300,3 +300,7 @@
     padding: 2em;
     background-color: $light-gray;
 }
+
+.selectize-dropdown {
+    z-index: 200 !important;
```

Comment:
Maybe make this selector a little bit more specific? Otherwise you will be affecting all `.selectize-dropdown`, which may result in unintended consequences in other parts of the code.